core-bundle/contao/widgets/SerpPreview.php

```diff
@@ -34,6 +34,11 @@ class SerpPreview extends Widget
 	 */
 	public function generate()
 	{
+		if (Input::isPost())
+		{
+			return '<div class="serp-preview"><p class="tl_gerror">' . $GLOBALS['TL_LANG']['MSC']['noSerpPreviewPost'] . '</p></div>';
```

Comment:
```suggestion
			return '<div class="serp-preview"><p class="tl_info">' . $GLOBALS['TL_LANG']['MSC']['noSerpPreviewPost'] . '</p></div>';
```

I don't know if we need this feature at all, but if we want to add it, it should be an info message and not an error message.